libs/components/src/lib/alert/alert.scss

```
@@ -0,0 +1,75 @@
+@use "partials/variables" as variables;
+
+@use "../../../../../dist/libs/tokens/scss/tokens.constants" as constants;
+
+@use "../../../../shared/src/lib/sass/mixins/connotation/config" with (
+	$connotations: accent success information alert warning,
+	$shades: primary primary-text,
+	$default: accent
+);
+@use "../../../../shared/src/lib/sass/mixins/connotation" as connotation;
+@use "../../../../shared/src/lib/sass/mixins/appearance/config" as appearance-config with (
+	$appearances: filled,
+	$default: filled
```

```suggestion
	$appearances: ghost,
	$default: ghost
```
the icon are not filled but ghost, so I this this is better.
Just please double check that the design is still OK :)